components/EntryForm.js

```diff
@@ -0,0 +1,72 @@
+import styled from "styled-components";
+import DatePicker from "react-datepicker";
+import { useState } from "react";
+import "react-datepicker/dist/react-datepicker.css";
+
+export default function EntryForm() {
+  const [startDate, setStartDate] = useState(new Date());
+  const [selectedOption, setSelectedOption] = useState("Car");
```

Comment:
You may add a constant here which holds all options for your dropdown menu, something like
```js
  const options = ["Car","Plane","Train","Bicycle"];
```
That way you can map over the list and create the individual options for your `select` further down in your JSX.

You don't **need** to do that. It's just a cleaner way of organizing your data and displaying it. And `options` is data. Typically, you would read that data from somewhere (database, APIs, localStorage) and then display it in the JSX.